### ISSUE TITLE
Location tracking via external connection

### ISSUE THREAD

**Issue by maddock34** [AHAAAAAAA/PokemonGo-Map#3163]
I'm running the map and have allowed external access to it, which is allowing me to access it from my Android on the go, problem I have is I can't seem to get it to locate and follow my phone. Is it possible to have it running on my PC at the location I set and be able to access and track my phone independently from the same process/server? If not how would I go about doing this? Thanks

**Comment by miaojs** [comment]
For me it didn't work until I had it running over https

**Comment by maddock34** [comment]
So you weren't able to get it to track your phone until you got it running over HTTPS? How did you get it to run over HTTPS?

**Comment by miaojs** [comment]
Correct. Lowest effort way would probably be using cloudflare if you have a domain name. Otherwise maybe look into letsencrypt for a free ssl cert.

**Comment by florian-184-de** [comment]
Use Firefox. FF sends your location even without SSL/TSL

**Comment by maddock34** [comment]
I didn't realise it was as simple as switching browsers. Thanks
